src/types/Order.ts

```diff
@@ -0,0 +1,29 @@
+export interface Order {
+    orderId: String;
+    orderName: String;
```

Comment:
```suggestion
    orderName: string;
```

Change `String` to `string` in all type declaration

Reply:
Improved Sir